col/src/main/java/vct/col/origin/Blame.scala

```diff
@@ -277,6 +277,21 @@ sealed trait FrontendDerefError extends VerificationFailure
 sealed trait FrontendAdditiveError extends VerificationFailure
 sealed trait FrontendSubscriptError extends VerificationFailure
 
+case class PlusProviderNull(node: Node[_]) extends FrontendAdditiveError with NodeVerificationFailure {
+  override def code: String = "plusProviderNull"
+  override def descInContext: String = "This expression might be null, which is prohibited because it is the subject of a custom plus operation."
+  override def inlineDescWithSource(source: String): String = s"The expression in $source might be null, which is prohibited for a custom plus operation."
```

Comment:
Oh nooooo

Reply:
I've been thinking we should probably support (in ghost code anyway) the case where this may be null - I don't think there's a theoretical problem with it and it's occasionally practical (especially for functions).

Reply:
In short term, it's either that, or refactoring the current operator instance method to be an (implicitly static) method that takes two arguments, and where you are not allowed to use `this`.